kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java

```diff
@@ -219,10 +218,11 @@ public String getReason() {
   }
 
   /**
-   * HTTP response body converted to specified {@code type}. {@code null} if there is no response.
+   * HTTP error response body converted to specified {@code type}. {@code null} if there is no
```

Comment:
Let's keep the info about the return value in one place.
```suggestion
   * HTTP error response body converted to the specified {@code type}.
```

Reply:
Modified the javadoc statement to : HTTP error response body converted to the specified {@code type}